File: src/main/java/ca/joelathiessen/kaly2/subconscious/sensor/Kaly2PulsedLightLidarLiteV2.java
package ca.joelathiessen.kaly2.subconscious.sensor;

import lejos.hardware.port.Port;
import lejos.hardware.sensor.I2CSensor;

public class Kaly2PulsedLightLidarLiteV2 extends I2CSensor {

    public Kaly2PulsedLightLidarLiteV2(Port port) {
        super(port);
    }

    // TODO: install the sensor!
    public void fetchSample(float[] sample, int offset) {

    }
}

File: core/src/main/kotlin/ca/joelathiessen/kaly2/core/subconscious/sensor/Kaly2PulsedLightLidarLiteV2.kt
package ca.joelathiessen.kaly2.core.subconscious.sensor

import lejos.hardware.port.Port
import lejos.hardware.sensor.I2CSensor

class Kaly2PulsedLightLidarLiteV2(port: Port) : I2CSensor(port), Kaly2Sensor {

    // TODO: install the sensor!
    override fun fetchSample(sample: FloatArray, offset: Int) {
    }
}
